cubi_tk/sodar/pull_raw_data.py

```diff
@@ -171,26 +179,38 @@ def execute(self) -> typing.Optional[int]:
                     break
 
         library_to_folder = self._get_library_to_folder(assay)
+
         commands = self._build_commands(assay, library_to_folder)
         if not commands:
             logger.info("No samples to transfer with --min-batch=%d", self.config.min_batch)
             return 0
-        return self._executed_commands(commands)
```

Comment:
This seems out of place. I'd leave all this logic inside `_executed_commands()`, the rest of the code just need to know if fails or not.